### PR TITLE
refactor(app): replaced old font size with TYPOGRAPHY const

### PR DESCRIPTION
**Pull request by MichaelMulieri**
# Overview
Replaced old font size with TYPOGRAPHY const where applicable.  

# Changelog
-  Replaced old font sizes that have  mapping to an equivalent TYPOGRAPHY const. 
-  Removed imports of unused variables.
-  Included import of TYPOGRAPHY where needed.

# Review requests

# Risk assessment
Low

### CODE REVIEW

**Comment by vabruzzo** [comment]
Thank you for this contribution! it looks like there is a format error in `app/src/organisms/LabwarePositionCheck/IntroScreen.tsx`, and the title of this pr should be formatted properly, for this it should begin with `refactor(app): replaced`

**Comment by koji** [comment]
Hi @MichaelMulieri 
Seems that you need to run the following command to fix lint-js error.

```zsh
npx prettier --write app/src/organisms/LabwarePositionCheck/IntroScreen.tsx
```

**Review by vabruzzo — APPROVED** [review]
this looks good to me, thank you for your contribution Mike!

File: app/src/organisms/LabwarePositionCheck/StepDetailText.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const StepDetailText = (
       <Box
         padding={SPACING_2}
         justifyContent={JUSTIFY_CENTER}
-        fontSize={FONT_SIZE_BODY_2}
+        fontSize={TYPOGRAPHY.fontSizeCaption}
```
Comment (koji) [inline app/src/organisms/LabwarePositionCheck/StepDetailText.tsx]:
```suggestion
        fontSize={TYPOGRAPHY.fontSizeH3}
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 2]
```diff
@@ -78,22 +77,22 @@ export const ThermocyclerModuleData = (
           textTransform={TYPOGRAPHY.textTransformUppercase}
           color={COLORS.darkGreyEnabled}
           fontWeight={FONT_WEIGHT_REGULAR}
-          fontSize={FONT_SIZE_CAPTION}
+          fontSize={TYPOGRAPHY.fontSizeH3}
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
          fontSize={TYPOGRAPHY.fontSizeCaption}
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 3]
```diff
@@ -78,22 +77,22 @@ export const ThermocyclerModuleData = (
...
           marginTop={SPACING_2}
           marginBottom={SPACING.spacing2}
         >
           {t('tc_lid')}
         </Text>
         <Text
           title="lid_target_temp"
-          fontSize={FONT_SIZE_CAPTION}
+          fontSize={TYPOGRAPHY.fontSizeH3}
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
          fontSize={TYPOGRAPHY.fontSizeCaption}
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 4]
```diff
@@ -78,22 +77,22 @@ export const ThermocyclerModuleData = (
...
           marginBottom={SPACING.spacing1}
         >
           {t(lidTarget == null ? 'na_temp' : 'target_temp', {
             temp: lidTarget,
           })}
         </Text>
-        <Text title="lid_temp" fontSize={FONT_SIZE_CAPTION}>
+        <Text title="lid_temp" fontSize={TYPOGRAPHY.fontSizeH3}>
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
        <Text title="lid_temp" fontSize={TYPOGRAPHY.fontSizeCaption}>
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 5]
```diff
@@ -106,22 +105,22 @@ export const ThermocyclerModuleData = (
           textTransform={TYPOGRAPHY.textTransformUppercase}
           color={COLORS.darkGreyEnabled}
           fontWeight={FONT_WEIGHT_REGULAR}
-          fontSize={FONT_SIZE_CAPTION}
+          fontSize={TYPOGRAPHY.fontSizeH3}
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
          fontSize={TYPOGRAPHY.fontSizeCaption}
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 6]
```diff
@@ -106,22 +105,22 @@ export const ThermocyclerModuleData = (
...
           marginTop={SPACING_2}
         >
           {t('tc_block')}
         </Text>
         <StatusLabel status={status} {...getStatusLabelProps(status)} />
         <Text
           title="tc_target_temp"
-          fontSize={FONT_SIZE_CAPTION}
+          fontSize={TYPOGRAPHY.fontSizeH3}
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
          fontSize={TYPOGRAPHY.fontSizeCaption}
```

File: app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx [hunk 7]
```diff
@@ -106,22 +105,22 @@ export const ThermocyclerModuleData = (
...
           marginBottom={SPACING.spacing1}
         >
           {t(targetTemp == null ? 'na_temp' : 'target_temp', {
             temp: targetTemp,
           })}
         </Text>
-        <Text title="tc_current_temp" fontSize={FONT_SIZE_CAPTION}>
+        <Text title="tc_current_temp" fontSize={TYPOGRAPHY.fontSizeH3}>
```
Comment (koji) [inline app/src/organisms/ModuleCard/ThermocyclerModuleData.tsx]:
```suggestion
        <Text title="tc_current_temp" fontSize={TYPOGRAPHY.fontSizeCaption}>
```

**Review by koji — CHANGES_REQUESTED** [review]
@MichaelMulieri, thank you for your contribution!
I found some misconverting of the font sizes and I left comments.
Please take a look!

**Comment by MichaelMulieri** [comment]
> @MichaelMulieri, thank you for your contribution! I found some misconverting of the font sizes and I left comments. Please take a look!

Thanks @koji.  I'll check it out.

File: app/src/organisms/LabwarePositionCheck/SectionList.tsx [hunk 8]
```diff
@@ -39,7 +39,7 @@ export function SectionList(props: Props): JSX.Element {
 
   return (
     <Box
-      fontSize={FONT_SIZE_CAPTION}
+      fontSize={TYPOGRAPHY.fontSizeH3}
```
Comment (koji) [inline app/src/organisms/LabwarePositionCheck/SectionList.tsx]:
```suggestion
      fontSize={TYPOGRAPHY.fontSizeCaption}
```

**Review by koji — CHANGES_REQUESTED** [review]
Need one more change

**Review by koji — APPROVED** [review]
The changes look good to me.
@MichaelMulieri, thank you for your contribution 🚀🚀🚀

**Comment by MichaelMulieri** [comment]
> The changes look good to me. @MichaelMulieri, thank you for your contribution 🚀🚀🚀

@koji Excellent. Thank you!

**Pull request merged**